pkg/util/fxutil/fxoptional/component.go

```diff
@@ -0,0 +1,97 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package fxoptional
```

Comment:
Would it be possible to add tests for this new code?

Reply:
The code in this file is just a wrapper to have more meaningful names. There are just aliases and don't contain any logic except the one for  `optional.NewOption` which is already unit tested. I don't know what to unit test? What do you think?

Reply:
We discussed this over Zoom. There is no need to add a unit in this case, as any change in the semantics of `fxoptional `and `optional.Option` would be caught by the compiler 😄